graylog2-web-interface/src/theme/index.js

```diff
@@ -14,12 +16,20 @@ const theme: ThemeInterface = {
 
 const themeModes: Array<string> = ['teinte'];
 
+const themePropTypes = PropTypes.shape({
+  breakpoints: PropTypes.object,
+  colors: PropTypes.object,
+  fonts: PropTypes.object,
+  utils: PropTypes.object,
+});
```

Comment:
Maybe something for a separate PR, but we can also use this prop type in `GenericPlot`, `SavedSearchControls` and `ColorLabel`

Reply:
I can update them in this PR if you think it's useful to do now. Otherwise it can be a follow-up PR.

Reply:
Lets do this in  follow-up PR.